lcls_tools/data_analysis/archiver.py

```diff
@@ -191,35 +352,45 @@ def side_effect(pvList, time):
+                                            u'BEND:LTUH:220:BDES': [10.6999430312505]})
             elif time == datetime(2020, 10, 8, 11, 56, 30, 10):
-                return {u'BEND:LTUH:280:BDES': 10.69994225003475,
-                        u'BEND:LTUH:220:BDES': 10.6999430312505}
+                return ArchiverData(timeStamps=[datetime(2020, 10, 8, 11, 56, 30, 10)],
+                                    values={u'BEND:LTUH:280:BDES': [10.69994225003475],
+                                            u'BEND:LTUH:220:BDES': [10.6999430312505]})
             elif time == datetime(2020, 10, 9, 11, 56, 30, 10):
-                return {u'BEND:LTUH:280:BDES': 10.69998960449891,
-                        u'BEND:LTUH:220:BDES': 10.699989622852124}
+                return ArchiverData(timeStamps=[datetime(2020, 10, 9, 11, 56, 30, 10)],
+                                    values={u'BEND:LTUH:280:BDES': [10.69998960449891],
+                                            u'BEND:LTUH:220:BDES': [10.699989622852124]})
             elif time == datetime(2020, 10, 10, 11, 56, 30, 10):
-                return {u'BEND:LTUH:280:BDES': 10.709996462228325,
-                        u'BEND:LTUH:220:BDES': 10.709996502988615}
+                return ArchiverData(timeStamps=[datetime(2020, 10, 10, 11, 56, 30, 10)],
+                                    values={u'BEND:LTUH:280:BDES': [10.709996462228325],
+                                            u'BEND:LTUH:220:BDES': [10.709996502988615]})
             elif time == datetime(2020, 10, 11, 11, 56, 30, 10):
-                return {u'BEND:LTUH:280:BDES': 10.709943385505909,
-                        u'BEND:LTUH:220:BDES': 10.709944052738756}
+                return ArchiverData(timeStamps=[datetime(2020, 10, 11, 11, 56, 30, 10)],
+                                    values={u'BEND:LTUH:280:BDES': [10.709943385505909],
+                                            u'BEND:LTUH:220:BDES': [10.709944052738756]})
             elif time == datetime(2020, 10, 12, 11, 56, 30, 10):
-                return {u'BEND:LTUH:280:BDES': 10.709999116032312,
-                        u'BEND:LTUH:220:BDES': 10.709999125516292}
+                return ArchiverData(timeStamps=[datetime(2020, 10, 12, 11, 56, 30, 10)],
+                                    values={u'BEND:LTUH:280:BDES': [10.709999116032312],
+                                            u'BEND:LTUH:220:BDES': [10.709999125516292]})
             elif time == datetime(2020, 10, 13, 11, 56, 30, 10):
-                return {u'BEND:LTUH:280:BDES': 10.70997130324195,
-                        u'BEND:LTUH:220:BDES': 10.709944052743765}
+                return ArchiverData(timeStamps=[datetime(2020, 10, 13, 11, 56, 30, 10)],
+                                    values={u'BEND:LTUH:280:BDES': [10.70997130324195],
+                                            u'BEND:LTUH:220:BDES': [10.709944052743765]})
             else:
```

Comment:
Why the switch/addition of this time format  'datetime(2020, 10, 13, 11, 56, 30, 10)' in these cases? Was 10.70997130324195 from before not enough?

Reply:
No the elif doesn't bother me, I was wondering about the format for times here. Did the format change or am I misunderstanding this?